src/libsyntax/ast.rs

```diff
@@ -1287,6 +1291,14 @@ pub enum Movability {
     Movable,
 }
 
+/// Whether an `await` comes from `await!` or `.await` syntax.
+/// FIXME: this should be removed when support for legacy `await!` is removed.
```

Comment:
Can you file an issue for that removal and reference it here?

Reply:
https://github.com/rust-lang/rust/issues/60610

Reply:
Done.